app/src/main/java/fr/free/nrw/commons/explore/paging/BasePagingFragment.kt

```diff
@@ -48,6 +48,13 @@ abstract class BasePagingFragment<T> : CommonsDaggerSupportFragment(),
         )
     }
 
+    override fun onConfigurationChanged(newConfig: Configuration) {
```

Comment:
Simple Javadoc: shortly say when this is called and what it does

Reply:
Done!